cmd/skynet/skynetcmd.go

```diff
@@ -70,6 +70,53 @@ fetch a skylink file from a chosen skynet portal.`,
 		Run: wrap(skynetpincmd),
 	}
 
+	skynetSkykeyCmd = &cobra.Command{
+		Use:   "skykey",
+		Short: "Perform skykey operations.",
+		// A subcommand must be provided.
+	}
+
+	skynetSkykeyAddCmd = &cobra.Command{
+		Use:   "add [skykey]",
+		Short: "Add a skykey to Skynet.",
+		Long:  "Store the given base-64 encoded skykey with the skykey manager.",
+		Run:   wrap(skynetaddskykeycmd),
+	}
+
+	skynetSkykeyCreateCmd = &cobra.Command{
+		Use:   "create [skykey name]",
+		Short: "Create a skykey on Skynet.",
+		Long:  "Returns a new skykey created and stored under that name.",
+		Run:   wrap(skynetcreateskykeycmd),
+	}
+
+	skynetSkykeyGetCmd = &cobra.Command{
+		Use:   "get",
+		Short: "Perform skykey get operations.",
+		// A subcommand must be provided.
+	}
+
+	skynetSkykeyGetIDCmd = &cobra.Command{
+		Use:   "id [skykey id]",
+		Short: "Get a skykey given its ID.",
+		Long:  "Get a skykey given its ID.",
+		Run:   wrap(skynetgetskykeyidcmd),
+	}
+
+	skynetSkykeyGetNameCmd = &cobra.Command{
+		Use:   "name [skykey name]",
+		Short: "Get a skykey given its name.",
+		Long:  "Get a skykey given its name.",
+		Run:   wrap(skynetgetskykeynamecmd),
+	}
+
+	skynetSkykeyGetSkykeysCmd = &cobra.Command{
+		Use:   "skykeys",
```

Comment:
This means someone has to type `skynet skykey get skykeys`?  Personal preference would be to just have `skynet skykey list`.

Reply:
Yeah that sounds better